cmd/token.go

```diff
@@ -142,7 +155,17 @@ The OIDC token includes claims regarding the GCE VM, which is verified by Attest
 			}
 		}
 
+		if cloudLogEnabled {
+			cloudLogger.Log(logging.Entry{Payload: map[string]string{"token": string(token)}})
```

Comment:
Logging the token seems like it could be dangerous, as it can be used as a bearer token. I thought we were only logging the attestation message

Reply:
Please PM me the PDD

Reply:
As we discussed, I think that we should integrate with custom audience and require a custom audience for cloud logging. This ensures that auditors don't accidentally mint a bearer token that can be used to access other cloud resources.